.circleci/codecov.yml

```diff
@@ -0,0 +1,30 @@
+coverage:
+  status:
+    project:
+      default: off
+      unittests:
```

Comment:
Since this is enforced by maven, you maybe don't need it here in codecov, But you can also leave it, at least I learned about the `flags` property. ;-)